.github/workflows/ci.yml

```diff
@@ -0,0 +1,212 @@
+name: Node CI Suite
+
+on:
+  push
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
+        os: [ubuntu-latest, macos-latest]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: yarn --frozen-lockfile --network-timeout 1000000
+      - run: yarn test
+
+  acceptance:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    environment: AcceptanceTests
+    env:
+      RUN_ACCEPTANCE_TESTS: true
+      HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: yarn --frozen-lockfile
+      - run: ./bin/run whoami
+      - run: ./scripts/run-acceptance
+
+  pack_deb:
```

Comment:
Unable to fully test the script from this point on, as it only runs upon an actual release. Next time we do a release, if this fails, we may need to package and release manually until we can ensure this works.